MaxText/configs/base.yml

```diff
@@ -121,6 +121,15 @@ record_internal_nn_metrics: 0
 # Create a GCS bucket, e.g. my-maxtext-outputs and set this to "gs://my-maxtext-outputs/"
 base_output_directory: ""
 
+# Whether or not to enable emergency checkpoint. If True, `enable_emergency_checkpoint` must also be specified.
+enable_emergency_checkpoint: False
+
+# It should be speficied when and only when `enable_emergency_checkpoint` is True.
+local_checkpoint_directory: ""
+
+# It should be non-zero when and only when `enable_emergency_checkpoint` is True.
+local_checkpoint_period: 0
```

Comment:
I think it's better if options like this default to None to indicate disabled, but up to you

Reply:
I tried but prefer it this way to avoid unnecessarily convoluted arg parsing.